src/index.js

```diff
@@ -0,0 +1,178 @@
+let temperatureLabel = null;
+let temperatureIncreaseButton = null;
+let temperatureDecreaseButton = null;
+let getRealtimeTemperatureButton = null;
+let skyImage = null;
+let landscapeImage = null;
+let cityNameInput = null;
+let cityNameResetButton = null;
+let skySelectValue = null;
+let gardenBackground = null;
+let temperatureValue = null;
+let defaultCityName = null;
+
+const showTemp = () => {
+    temperatureLabel.textContent = temperatureValue;
+}; 
+
+const showCity = () => {
+    cityNameLabel.textContent = defaultCityName;
+    cityNameInput.value = defaultCityName;
+}; 
+
+const setTempLabelColor = (temperature) => {
+    if (temperature >= 80) {
+        return temperatureLabel.setAttribute("class", "red");
+    } else if (temperature >= 70) {
+        return temperatureLabel.setAttribute("class", "orange");
+    } else if (temperature >= 60) {
+        return temperatureLabel.setAttribute("class", "yellow");
+    } else if (temperature >= 50) {
+        return temperatureLabel.setAttribute("class", "green");
+    } else {
+        return temperatureLabel.setAttribute("class", "teal");
+    }
+};
+
+const landscapeImages = {
+    80: "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂",
+    70: "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷",
+    60: "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃",
+    50: "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲",
+};
```

Comment:
Rather than storing the landscape values keyed off the temperature, consider using a list. We don't actually need to look up the landscape by temperature very often (many different temperatures map to the same landscape). Rather, if the landscapes were stored in a list, we could store them in ascending or descending order, storing the top or bottom of the range (or both) along with the landscape in a tuple. Something like the following:

```py
const landscapeImages = [
    (80, "🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂", "red"),
    (70, "🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷", "orange"),
    (60, "🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃", "green"),
    (50, "🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲", "blue"),
];
```

Notice that we could also include the class name to apply to the temperature as well! Then, when the temperature is set, we could iterate through the list to find the first record less than or equal to the current temperature. So if the temp was 100, we would match the 80 record. Whereas if the temp was 65, we would match the 60 record. If we find no matching record, we could take the final one.

From the matching record, we can determine both the landscape to use, as well as the class to apply to the temperature display, and could avoid brittle chains of `if`/`else if` checks.